docs/using.md

```diff
@@ -171,41 +489,294 @@ is passed in `infrastructure_provisioning: bar` and the third `arbitrary_key: ba
+    
+    UseMe:
+      Message: Hello Phase 2
+      Repeat: {^Parameter: {Name: "Repeat", Default: 1}}
+
+Using `LoadConfig`, the contents of the `UseMe` key will be placed into the location where the `LoadConfig` was evaluated, with parameters substituted, so we end up with the following ouput from evaluation: 
+
+    Actors:
+    - Name: HelloWorld
+      Type: HelloWorld
+      Threads: 2
+      Phases:
+      - Message: Hello Phase 0 🐳
+        Duration: 50 milliseconds
+      - Message: Hello Phase 2
+        Repeat: 2
+
+A few notes:
+
+-   The parameter `Repeat` was substituted in. A loaded config can have any number of parameters substituted, at any key's value.
+-   The loaded config filepath should be relative to the location of the workload containing `LoadConfig`.
+-   The contents of the loaded config are shallow-merged into the location where the `LoadConfig` is evaluated. If there are conflicting keys at the location, the existing key-values are kept. There is no deep dict merge at present.
+
+The `LoadConfig` keyword can be used to substitute and parameterize anything, including entire workloads! For an example of this, see [here](../src/workloads/docs/HelloWorld-LoadConfig.yml).
+
+
+<a id="orga45b1d8"></a>
+
+### ActorTemplate
+
+Genny also offers a syntax for templatizing actors. This is useful if there are many actors that share common configuration, which need to differ in specific ways. An example of this can be found [here](../src/workloads/docs/HelloWorld-ActorTemplate.yml).
+
+
+<a id="orgf9c328f"></a>
+
+### OnlyActiveInPhases
+
+If there are many phases, and an actor only needs to run for some of them, there is an alternative syntax to specify only the phases the actor runs in. Consider the following actor:
+
+    Actors:
+    - Name: HelloWorld
+      Type: HelloWorld
+      Threads: 2
+      Phases:
+      OnlyActiveInPhases:
+        Active: [0, 2]
+        NopInPhasesUpTo: 3
+        PhaseConfig:
+          Message: Alternate Phase 1
+          Repeat: 100
+
+This configures the actor to run with the given configuration in phases named 0 and 2, and nops in all other phases up phase named 3.
+
+
+<a id="org22b7a0f"></a>
+
+### Defaults and Overrides
+
+By default, a Genny workload yaml contains the following configuration:
+
+    Clients:
+      Default:
+        QueryOptions:
+          maxPoolSize: 100
+
+For more details about this configuration's purpose, see the section on [Connecting to the Server](#orgd6b0450).
+
+Genny also has an override syntax for configuring workloads. When invoking Genny, you can use the `-o` option to specify an override file.
+This uses [OmegaConf](<https://omegaconf.readthedocs.io/en/2.1_branch/>) to merge the override file onto the workload. This functionality
+should only be used to set values that absolutely need to be specified at runtime, such as URIs for systems under test. (See [Connecting to the Server](#orgd6b0450) for details.)
```

Comment:
Can we please please please not advertise this functionality? It is super super dangerous to have infinitely many ways to configure things, and I don't think we want to encourage anyone doing anything deep with omegaconf because then we're tied to it forever.

I think we'd need to re-evaluate some of the genny syntax before deciding for sure we want omegaconf or, alternatively, if we need preprocessing given we have omegaconf.

I won't insist on this of course, but we put a lot of care into genny config to make it at least _mostly_ sensible and absolutely not turingn complete (we're not trying to write a yaml driver over here).

Reply:
I'm also dubious about advertising the overrides file. It's a hack that we didn't have a better solution for. If we were designing the genny syntax from scratch today I doubt we would arrive at an overrides file given the requirements of those workloads (🔮 I think we'd have DSI output a connection string mapping or something and have "overrides" only apply to named conn).

So similar to notes about omegaconf, at the very least I would add a caveat emptor to come chat with us if you think you need overrides or omegaconf or any of the non-`^`-style preprocessor features.